### PR TITLE
Update version: zufuliu.notepad2 version 4.24.05r5242

### PR DESCRIPTION
**Pull request by tianyeeT**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/157927)

### CODE REVIEW

**Comment by tianyeeT** [comment]
@microsoft-github-policy-service agree

File: manifests/z/zufuliu/notepad2/4.24.05r5242/zufuliu.notepad2.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: zufuliu.notepad2
+PackageVersion: 4.24.05r5242
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: portable
```
Comment (tianyeeT) [inline manifests/z/zufuliu/notepad2/4.24.05r5242/zufuliu.notepad2.installer.yaml:8]:
```suggestion
```

File: manifests/z/zufuliu/notepad2/4.24.05r5242/zufuliu.notepad2.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- htm
+- html
+- json
+- log
+- md
+- txt
+ReleaseDate: 2024-05-19
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/zufuliu/notepad4/releases/download/v4.24.05r5242/Notepad2_en_ARM64_v4.24.05r5242.zip
```
Comment (tianyeeT) [inline manifests/z/zufuliu/notepad2/4.24.05r5242/zufuliu.notepad2.installer.yaml:21]:
```suggestion
- Architecture: x86
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: notepad2.exe
  InstallerUrl: https://github.com/zufuliu/notepad4/releases/download/v4.24.05r5242/Notepad2_en_ARM64_v4.24.05r5242.zip

```

File: manifests/z/zufuliu/notepad2/4.24.05r5242/zufuliu.notepad2.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  InstallerSha256: F6C60B44960F0A493E0D84992FFBEDB2E9BAB0A08636E8DF854422D1D7C6E61D
+- Architecture: x64
+  InstallerUrl: https://github.com/zufuliu/notepad4/releases/download/v4.24.05r5242/Notepad2_en_x64_v4.24.05r5242.zip
```
Comment (tianyeeT) [inline manifests/z/zufuliu/notepad2/4.24.05r5242/zufuliu.notepad2.installer.yaml:24]:
```suggestion
- Architecture: x64
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: notepad2.exe
  InstallerUrl: https://github.com/zufuliu/notepad4/releases/download/v4.24.05r5242/Notepad2_en_x64_v4.24.05r5242.zip

```

File: manifests/z/zufuliu/notepad2/4.24.05r5242/zufuliu.notepad2.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  InstallerSha256: 9EB8981657C588701A7493A8DD0DEB2A4F578BF921ADFE7682B467D2068368B3
+- Architecture: arm
+  InstallerUrl: https://github.com/zufuliu/notepad4/releases/download/v4.24.05r5242/Notepad2_en_ARM64_v4.24.05r5242.zip
```
Comment (tianyeeT) [inline manifests/z/zufuliu/notepad2/4.24.05r5242/zufuliu.notepad2.installer.yaml:27]:
```suggestion
- Architecture: arm
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: notepad2.exe
  InstallerUrl: https://github.com/zufuliu/notepad4/releases/download/v4.24.05r5242/Notepad2_en_ARM64_v4.24.05r5242.zip

```

File: manifests/z/zufuliu/notepad2/4.24.05r5242/zufuliu.notepad2.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  InstallerSha256: F6C60B44960F0A493E0D84992FFBEDB2E9BAB0A08636E8DF854422D1D7C6E61D
+- Architecture: arm64
+  InstallerUrl: https://github.com/zufuliu/notepad4/releases/download/v4.24.05r5242/Notepad2_en_ARM64_v4.24.05r5242.zip
```
Comment (tianyeeT) [inline manifests/z/zufuliu/notepad2/4.24.05r5242/zufuliu.notepad2.installer.yaml:30]:
```suggestion
- Architecture: arm64
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: notepad2.exe
  InstallerUrl: https://github.com/zufuliu/notepad4/releases/download/v4.24.05r5242/Notepad2_en_ARM64_v4.24.05r5242.zip

```

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 885.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**